.github/workflows/release_centos8.yaml

```diff
@@ -190,7 +190,7 @@ jobs:
 
           # Build release binaries.
           cd build
-          cmake -DCMAKE_BUILD_TYPE=release ..
+          cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```

Comment:
The same mode as we ship to the downloads -> Release at the moment, config

Reply:
As I understand, we can make configurable build here only if make build type as option as for package_all. But we can't sync package_all option with this one I believe.